src/openmrs-backend-dependencies.ts

```diff
@@ -0,0 +1,4 @@
+export const omrsBackendDependencies = {
+  "webservices.rest": "2.24.0",
```

Comment:
Is there any reason why we did not use the latest released versions of these modules?

Reply:
Good question -- I think we should actually seek to find the **oldest** version possible that will work for what we need and then do `"webservices.rest": ">=2.<oldestthingthatworks>"` (or maybe `^` instead of `>=`). That would make this frontend module not ever complain when the backend modules are fine.

As for why it's 2.24.0 instead of latest -- my guess is that that was just an accident. We probably 2.24.0 on openmrs-spa.org and didn't realize that we were behind. @fatmali can you confirm?

Reply:
I like the suggestion of minimum required version. It seems to be in line with the `require_version` here https://wiki.openmrs.org/display/docs/Module+Config+File

Reply:
👍